Allow setting the pooling time in DLPHook

xuan616 opened the pull request:
Issue link: #8822 
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

xuan616 committed:
Make 'time_to_sleep_in_seconds' configurable by setting it as the parameter in create_dlp_job()

mik-laj commented on airflow/providers/google/cloud/hooks/dlp.py:
```diff
@@ -180,6 +178,7 @@ def create_dlp_job(
         timeout: Optional[float] = None,
         metadata: Optional[Sequence[Tuple[str, str]]] = None,
         wait_until_finished: bool = True,
+        time_to_sleep_in_seconds: int = 1
```
```suggestion
        time_to_sleep_in_seconds: int = 60
```

xuan616 replied:
Thanks, 60 helps a lot

mik-laj commented on airflow/providers/google/cloud/hooks/dlp.py:
```diff
@@ -207,6 +206,9 @@ def create_dlp_job(
         :type wait_until_finished: bool
         :rtype: google.cloud.dlp_v2.types.DlpJob
+        :param time_to_sleep_in_seconds: (Optional) Time to sleep between active checks
+            of the operation results. Defaults to 1.
```
```suggestion
            of the operation results. Defaults to 60s.
```

xuan616 replied:
Thanks for the suggested change.

ashb commented on airflow/providers/google/cloud/hooks/dlp.py:
```diff
@@ -207,6 +206,9 @@ def create_dlp_job(
         :type wait_until_finished: bool
         :rtype: google.cloud.dlp_v2.types.DlpJob
+        :param time_to_sleep_in_seconds: (Optional) Time to sleep between active checks
+            of the operation results. Defaults to 1.
```
```suggestion
        :param time_to_sleep_in_seconds: (Optional) Time to sleep, in seconds, between active checks
            of the operation results. Defaults to 60.
```

ashb replied:
(yes, it's in the name already).

If we say "default to 60s" then someone may try to put `"5m" `, or another string value there if they aren't paying close attention

xuan616 replied:
Thanks, the new docstring will dispel the potential confusion.

xuan616 committed:
Update airflow/providers/google/cloud/hooks/dlp.py

Change the default value of 'time_to_sleep_in_seconds' to 60

Co-authored-by: Kamil Breguła <mik-laj@users.noreply.github.com>

xuan616 committed:
Update airflow/providers/google/cloud/hooks/dlp.py

Update docstring according to the changed default value of 'time_to_sleep_in_seconds'

Co-authored-by: Ash Berlin-Taylor <ash_github@firemirror.com>
